static/src/js/components/EDSCImage/EDSCImage.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import Spinner from '../Spinner/Spinner'
+
+import './EDSCImage.scss'
+
+/**
+ * Renders EDSCImage.
+ * @param {Object} props - The props passed into the component.
+ * @param {String} props.alt - The alt text to be used as the alt attribute on the image.
+ * @param {String} props.className - An optional css class attribute.
+ * @param {String} props.height - The height of the image.
+ * @param {String} props.src - The src to be used as the src attribute on the image..
+ * @param {String} props.srcSet - The srcSet to be used as the srcSet attribute on the image.
+ * @param {String} props.width - The width of the imag.
```

Comment:
```suggestion
 * @param {String} props.width - The width of the image.
```

Reply:
Nice catch. Surprised this slipped by the spell checker :shaking-fist: